.azure/pipelines/ci.yml

```diff
@@ -29,7 +29,31 @@ variables:
 - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
   - name: _BuildArgs
     value: ''
+
 jobs:
+# Build Web.JS Linux x64
+- template: jobs/default-build.yml
+  parameters:
+    codeSign: true
+    jobName: WebJS_Linux_x64_build
+    jobDisplayName: "Build: Web.JS Linux x64"
```

Comment:
minor i.e. up to you: I suggest minimizing these names to `WebJS_build` and `Build: Web.JS` because the platform is an implementation detail here